src/tests/SortingProducts.spec.js

```diff
@@ -0,0 +1,50 @@
+import { expect } from '@playwright/test';
+import { test } from '../fixtures/base';
+
+test.beforeEach(async ({ app }) => {
+    await app.login.navigate();
+    await app.login.performLogin('standard_user', 'secret_sauce');
+
+    await expect(app.page).toHaveURL(/.*\/inventory.html/);
+
+    const itemCount = await app.inventory.inventoryItems.count();
+    expect(itemCount).not.toBe(0);
+});
+
+test.describe('Saucedemo tests for sorting products on invertory page', () => {
+    test('Sorting products by name A-Z', async ({ app }) => {
```

Comment:
One you create one method to perform sort - you will be able to have 2 groups of tests: 
- sort by name (az/za)
- sort by price (hilo/lohi)
These groups will contain 2 parametrised tests for each group - https://playwright.dev/docs/test-parameterize